src/app/shared/components/thumbnail/thumbnail.component.ts

```diff
@@ -199,4 +206,15 @@ export class ThumbnailComponent
       imageLoader.src = imageUrl;
     }
   }
+
+  chooseFullSizeImage(record: RecordVO) {
+    if (record.FileVOs.length > 1) {
+      const convertedUrl = record.FileVOs.find(
+        (file) => file.format == 'file.format.converted'
+      ).fileURL;
+      return convertedUrl;
+    } else {
+      return record.FileVOs[0].fileURL;
```

Comment:
```suggestion
      return record.FileVOs[0]?.fileURL;
```
This is pretty minor, but if a file is still processing it won't have any FileVOs at all and this line will throw an error. (I tested this by just uploading a bigger file and immediately clicking into it.) Though if we do add this, I think it might also be useful to return from `ngOnInit` if `chooseFullSizeImage()` returns an undefined value so that OpenSeaDragon doesn't need to be loaded in with a null URL.

Reply:
Thanks!